multiregion/docs/multiregion.rst

```diff
@@ -556,6 +572,9 @@ There is a script you can run to collect the JMX metrics from the command line,
       multi-region-async-op-leader-is-observer: 0
       multi-region-default: 0
 
+#. Metrics relating to partitions, followers and observers are viewable in the "Topics" section of the C3 UI.
```

Comment:
C3 doesn't show exactly the same JMX metrics as the output above.  Is the screenshot needed at all if this is the same diagram as shown earlier?

Reply:
It has been removed and the latest docs preview link reflects that.